kubevirtci

```diff
@@ -309,6 +324,37 @@ function kubevirtci::kubectl_tenant {
     kubectl --kubeconfig .${TENANT_CLUSTER_NAME}-kubeconfig --insecure-skip-tls-verify --server https://localhost:64443 "$@"
 }
 
+function kubevirtci::retry_until_success {
+    set +e
+    while true; do
+        sleep 5
+        $@
+        ret=$?
+        if [[ "$ret" == "0" ]]; then
+            break
+        fi
+    done
+    set -e
+}
+
+function kubevirtci::wait_for_vm {
+    local vm_name=$1
+    while true ; do
+		vms_list=$(${_kubectl} get vm -n ${TENANT_CLUSTER_NAMESPACE} --no-headers -o custom-columns=":metadata.name")
```

Comment:
@qinqon Hi. Just making sure, your intention here was to watch for VM or VMI to be created? The message `echo "Waiting for worker VMI in tenant cluster namespace"` refers to VMI

Reply:
Message is wrong, will fix it.